apps/communications/contacts/js/contacts.js

```diff
@@ -893,6 +888,9 @@ var Contacts = (function() {
     window.removeEventListener('DOMContentLoaded', onLoad);
   });
 
+  window.addEventListener('overlayshown', onOverlayShown);
+  window.addEventListener('overlayhidden', onOverlayHidden);
```

Comment:
Basically, due to `bulkDelete` has a `done` callback that can be executed [1]. Imagine that in the future we have Settings in a separate view. No event will be listened in `List`, so this approach probably needs to move to a Promise based one.


[1] https://github.com/mozilla-b2g/gaia/blob/master/apps/communications/contacts/js/views/settings.js#L245